api/client/volume/client.go

```diff
@@ -4,12 +4,13 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
-	"github.com/libopenstorage/openstorage/api"
-	"github.com/libopenstorage/openstorage/api/client"
-	"github.com/libopenstorage/openstorage/volume"
 	"io"
 	"io/ioutil"
 	"strconv"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/api/client"
+	"github.com/libopenstorage/openstorage/volume"
```

Comment:
Remove this file from the PR

Reply:
done, will send new PR today if I can close with everyone on their requests